lnrpc/rpc.proto

```diff
@@ -141,7 +141,24 @@ message UnlockWalletRequest {
 }
 message UnlockWalletResponse {}
 
+message ChangePasswordRequest {
+    bytes current_password = 1;
+    bytes new_password = 2;
+}
+message ChangePasswordResponse {}
+
 service Lightning {
+    /** lncli: `changepassword`
```

Comment:
A bit odd that this new method isn't on the `WalletUnlocker` service itself. This may be a worthy follow up, but would require us to _always_ run the unlocker service as an active RPC server. If we end up doing this, then we can also add an option to actively lock/unlock the wallet. This may be attractive for "box" type nodes that don't frequently forward, and only exist in order to carry out payments sourced from end end user.

Reply:
Fixed.